subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/testsuites/TestSuitesIntegrationTest.groovy

```diff
@@ -537,35 +473,6 @@ class TestSuitesIntegrationTest extends AbstractIntegrationSpec {
         succeeds("assertSameFrameworkInstance")
     }
 
-    def "multiple getTestingFramework() calls on a test suite return same instance even when calling useJUnit"() {
```

Comment:
Yeah, because the value is a mapping from the framework/version to a `VersionedTestingFramework` object.  Previously, this passed because the first call to `get()` finalized the property, so the second one got the finalized value.  Now, it returns a new `VersionedTestingFramework` with the same values every time.  We could fix this, but that object is just metadata, so this test case just didn't seem that important at all.